natlas-server/app/static/js/natlas-tagging.js

```diff
@@ -0,0 +1,36 @@
+$('#tagmodal').on('show.bs.modal', function (event) {
+  var button = $(event.relatedTarget) // Button that triggered the modal
+  var scopeid = button.data('scopeid') // Extract info from data-* attributes
+  var scopetarget = button.data('scopetarget') // Extract info from data-* attributes
+  var tagaction = button.data('action')
+  var tagstr = $('#scopeTags-'+scopeid).text().trim() // Trim whitespace in jinja formatting leaves whitespace
+  var tags = tagstr.split(', ')
+  var modal = $(this)
+  if (tagaction == 'add') {
+    modal.find('.modal-title').text('Add tag to ' + scopetarget)
+    modal.find('#tagScopeForm').attr('action', "/admin/scope/"+scopeid+"/tag")
+    modal.find('#tagScopeSubmit').text('Add Tag')
+    if (tagstr != ''){
+      tags.forEach(function(item) {
+        $("option[value=" + item.trim() + "]").attr('hidden', '') // hide this option because it's already added
+        $("option[value=" + item.trim() + "]").attr('disabled', '') // disable this option because it's hidden
```

Comment:
Thanks for pointing this out, it unveiled another bug that I need to resolve where the options don't get deleted/re-enabled when the modal shows again. This will probably not be a super likely thing to happen, but since I've noticed it, I'll also fix that.

Reply:
Switched to using `.attr({'hidden': '', 'disabled': ''})` as well as fixed other related bugs that this caused me to notice.